src/sbtc-releases/sbtc-dev/get-started.md

```diff
@@ -1,130 +1,22 @@
 # Get Started with sBTC 0.1
 
-## Local setup
-### Requirements
-* install [clarinet](https://github.com/hirosystems/clarinet)
-* checkout main branch of repository [github.com/stacks-network/sbtc](https://github.com/stacks-network/sbtc)
+The sBTC contract has been deployed on the Stacks testnet as *TODO contract id*.
```

Comment:
There are a lot of open issues that are TODOs in the published repo. Do you think it makes more sense to open an issue for that?

I am fine with not merging, however, we might not share enough knowledge between us.

Reply:
I don't think it adds much value anyway. leaving a TODO stub does not make me more or less knowledgable. I would prefer opening an issue or a ticket/task somewhere in the project management board. Consider adding a a short description and optionally tagging interested parties.

Instead of leaving a todo stub consider giving a broad truthy proposition instead.
If you can't state what the contract id is, consider describing what you expect to see or how you would look for it.

Reply:
Thanks for the input. I have removed the TODOs.